public/index.html

```diff
@@ -1,11 +1,15 @@
 <!doctype html>
-<html>
+<html itemscope itemtype="http://schema.org/Organization">
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
 
+  <meta itemprop="name" content="Hundred Zeros">
+  <meta itemprop="description" content="Home of the Hundred Zeros Destiny clan. PvE and PvP, Xbox and Playstation: we cover them all! Special emphasis on fun and camaraderie">
+  <meta itemprop="image" content="images/snippet.png">
```

Comment:
Is snippet.png the full size image?  If so, little nit.  Can we rename to something else more generic so it can be used on the site (i.e. destiny-banner-zeros.png...see what I did there? ;)